translations/expansion_content.tex/pl.po

```diff
@@ -141,6 +151,33 @@ msgid ""
 "\\end{minipage}\n"
 "\n"
 msgstr ""
+"\\begin{minipage}[t]{0.48\\textwidth}\n"
+"  \\centering\n"
+"  \\minipage[b]{0.5\\textwidth}\n"
+"    \\centering\n"
+"    \\includegraphics[width=0.75\\linewidth]{\\images/school_of_fire.png}\n"
+"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Szkoła Magii}}}\n"
```

Comment:
wg oryginału są to "Szkoła Magii Ognia" i tak dalej, z "Magii" w środku. może byłoby dobrze to odwzorować?

![Uploading Screenshot 2024-04-22 at 19.37.52.png…]()

Reply:
obraz mi sie nie wyswietla